.travis.yml

```
@@ -40,6 +41,32 @@ stages:
 
 jobs:
   include:
+  # opensuse: "recent" systems with ICB
+  #   note: when you PR, docker-cp provides, in the container, the branch associated with the PR (not master where there's nothing new)
+  #         1. docker create --name mobydick IMAGE CMD        <=> create a container (= instance of image) but container is NOT yet started
+  #         2. docker cp -a ${TRAVIS_BUILD_DIR} mobydick:/tmp <=> copy git repository (CI worker, checkout-ed on PR branch) into the container
+  #                                                               note: docker-cp works only if copy from/to containers (not images)
+  #         3. docker start -a mobydick                       <=> start to run the container (initialized with docker-cp)
+  - stage: opensuse
+    dist: bionic
+    script: |
+      sudo docker pull opensuse/tumbleweed                                   \
+      &&                                                                     \
+      sudo docker create --name mobydick opensuse/tumbleweed /bin/bash -c    \
+      "zypper install -y git gcc gcc-fortran gcc-c++ openmpi-devel        && \
+       zypper install -y cmake                                            && \
+       zypper install -y blas-devel lapack-devel                          && \
+       cd /tmp                                                            && \
+       cd arpack-ng                                                       && \
+       git status                                                         && \
+       git log -2                                                         && \
+       mkdir -p build && cd build                                         && \
+       cmake -DEXAMPLES=ON -DMPI=ON -DICB=ON ..                           && \
+       make all && make test"                                                \
```

```suggestion
       export PATH="/usr/lib64/mpi/gcc/openmpi2/bin:/usr/lib/mpi/gcc/openmpi2/bin/:$PATH" && \
       make all && make test"                                                \
```

`mpirun` exists in an openmpi-flavour specific location in lib (lib64 for 64-bit), so you will need to add it to `$PATH`.